src/main/java/com/softserve/sportshub/team/TeamController.java

```diff
@@ -0,0 +1,28 @@
+package com.softserve.sportshub.team;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/teams")
+public class TeamController {
+
+    @Autowired
+    private TeamService teamService;
+
+    @Autowired
+    private TeamTeamDtoMapper mapper;
+
+    @GetMapping("/")
+    public List<TeamDto> getTeams() {
+        List<Team> teams = teamService.findAll();
+
+        return mapper.teamListToTeamListDto(teams);
```

Comment:
You most likely shouldn't leak entities into controllers. Services should return dtos